docs/api/datetime.md

```diff
@@ -176,19 +179,35 @@ Be sure to check the [Browser Compatibility Chart](https://developer.mozilla.org
 
 ## Presentation
 
+By default, `ion-datetime` allows users to select both date and time. In addition, users have access to selecting the specific month, year, hour, and minute.
+
 Some use cases may call for only date selection or only time selection. The `presentation` property allows you to specify which pickers to show and the order to show them in.
 
+You can also control the order in which pieces of the datetime appear. For example, setting `date-time` will have the calendar picker appear before the time picker. Setting `time-date` will have the calendar picker appear after the time picker.
+
 ### Month and Year Selection
 
-TODO
+Month and year selection is available by passing `month-year`, `year-month`, `month`, or `year` to the `presentation` property.
+
+This example shows a datetime with the `month-year` configuration.
+
+<MonthAndYear />
```

Comment:
The height here triggers a scroll bar on `md` mode ([screenshot](https://i.gyazo.com/4ac8e724fb068f707e17ddee3628532c.png)). Could we make it a little taller?